File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int left=0;
        int right=numbers.length-1;
        int[] ar =new int[2];
        while(left<right){
            int midvalue=numbers[left]+numbers[right];
            if(midvalue==target){
                ar[0]=left+1;
                ar[1]=right+1;
                break;
            }else if(target<midvalue){
                right-=1;
            }else{
                left+=1;
                
            }
            
        }  
        return ar; 

        
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        list1=[]
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j] == target and i!=j:
                    list1.append(i)
                    list1.append(j)
                    return list1
                    break

        